submissions/kt-std/tiny-js-world/index.js

```diff
@@ -0,0 +1,57 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+   
+   function Inhabitant (species, name, gender, hands, legs, sayings, friends){
+   		this.species = species;
+   		this.name = name;
+   		this.gender = gender;
+   		this.hands = hands === 0 ? `${this.species}s don't have hands! They have paws`: hands;
+   		this.legs = legs;
+   		this.friends = Array.isArray(friends) && friends.length ? friends : 'Forever alone...';
+   		this.saySomething = sayings;
+   }
+
+   const dog = new Inhabitant('dog', 'Rex', 'male', 0, 4, 'Woof!', ['Mukhtar', 'Sharik', 'Thunder']);
+   const cat = new Inhabitant('cat', 'Murka', 'female', 0, 4, 'Meow!', ['Murchik', 'Anna']);
+   const woman = new Inhabitant('woman', 'Anna', 'female', 2, 2, 'Hi, folks!', ['John', 'Tom']);
+   const man = new Inhabitant('man', 'John', 'male', 2, 2, 'Ahoy!');
+   const catWoman = new Inhabitant('superhero', 'Selina', 'female?', 2, 2, cat.saySomething, ['Alice', 'Arizona']);
+
+   ((array) => {
+	   array.forEach( obj => {
+	   	let string = '';
+	   	for(prop in obj){
+	   		Array.isArray(obj[prop])? string += `${obj[prop].join(", ")}; ` : string += `${obj[prop]}; `;
+	   	}
+	   	print(string);
+	   });
+   })([dog, cat, woman, man, catWoman]);
```

Comment:
Why do you need an IIFE here?

Reply:
Thank you, @OleksiyRudenko, for your time and for reviewing this!
It is not needed here. It was used just for shortening the code a bit and for the avoidance of global namespace pollution as the function returns nothing (I felt like a C programmer for a while  :sweat_smile:). Waiting for your reply to understand whether my reasoning is correct or it is not. I changed it to the function declaration in the updated version of the code, as well as fixed improper code formatting. I reformatted it via JSPrettier and ESLint due to the Google-style preset. I've also added some js-docs as ESLint was failing with errors. I hope it looks just how it has to. 
Finally, I calmed down with message: `✓ 0 errors and warnings`.
BTW, lots of thanks for providing info about such useful plugins!